src/ray/core_worker/core_worker.h

```diff
@@ -1044,6 +1044,11 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   /// Return true if the core worker is in the exit process.
   bool IsExiting() const;
 
+  /// Retrieve the current statistics about tasks being received and executing.
+  /// \return an unordered_map mapping function name to list of (num_received,
+  /// num_executing, num_executed).
+  std::unordered_map<std::string, std::vector<size_t>> GetInflightTasksCount() const;
```

Comment:
absl::flat_hash_map

Reply:
We need to build Cython bridge here so unordered_map is way easier than absl.